Rubberduck.Main/Root/RubberduckIoCInstaller.cs

```diff
@@ -122,6 +124,13 @@ public void Install(IWindsorContainer container, IConfigurationStore store)
                 .ImplementedBy<VBEInteraction>()
                 .LifestyleSingleton());
 
+            container.Register(Component.For<IAddRemoveReferencesPresenterFactory>()
+                .ImplementedBy<AddRemoveReferencesPresenterFactory>()
+                .LifestyleSingleton());
```

Comment:
Since this is a registration for a concrete factory, shouldn't this be in the method that registers special factories?

Reply:
Good call - I completely missed that.